--- blog-topics/cets_blogtopics.php	2010-05-17 14:56:42.000000000 +0300
+++ blog-topics/cets_blogtopics.php	2010-05-17 14:56:42.000000000 +0300
@@ -534,7 +534,7 @@
     	}
 		
     	$blog_topic = $this->get_blog_topic($id);
-    	echo "<th>Blog Topic</th> <td><select name='blog_topic_id' id='blog_topic_id'>";
+    	echo "<th>",_e('Blog Topic'),"</th> <td><select name='blog_topic_id' id='blog_topic_id'>";
 		
     	foreach ( $this->get_topics() as $topic )  {
     		if ($blog_topic == $topic->id) {
@@ -553,7 +553,7 @@
 	function get_topics_select_signup($id = 0)
     {
     	
-    	echo "<label for='blog_topic_id'>Blog Topic:</label><select name='blog_topic_id' id='blog_topic_id'>";
+    	echo "<label for='blog_topic_id'>",_e('Blog Topic:'),"</label><select name='blog_topic_id' id='blog_topic_id'>";
 		
     	foreach ( $this->get_topics() as $topic )  {
 			echo "<option value='$topic->id' " .">" . $topic->topic_name ."</option>";
@@ -678,7 +678,7 @@
     {
     	global $wpdb;
     	
-    	echo "<table><tr><th>Topic ID</th><th valign='top'>Blog Topic:</th><th>Slug</th><th>Description<th>Edit/Add</th><th>Delete</th></tr>";
+    	echo "<table><tr><th>Topic ID</th><th valign='top'>",_e('Blog Topic:'),"</th><th>Slug</th><th>Description<th>Edit/Add</th><th>Delete</th></tr>";
     	foreach ( $this->get_topics() as $topic )  {
 			
     		echo "<tr><form name='catupdate' method='post'><td align='center'>" . $topic->id . "</td>";
@@ -709,7 +709,7 @@
 	// Adds the submenu to the blog settings screen
     function add_submenu()
     {
-    	add_options_page('Blog Topic Configuration', 'Blog Topic', 10, 'cets_blog_topic', array(&$this,'config_page'));
+    	add_options_page('Blog Topic Configuration', __('Blog Topic'), 10, 'cets_blog_topic', array(&$this,'config_page'));
     }
     
 	// Creates the configuration page for an individual blog (blog's settings screen sub menu)
@@ -742,8 +742,8 @@
         <div id="message" class="updated fade"><p><?php _e('Options saved.') ?></p></div>
         <?php	} ?>
         <div class="wrap">
-        <h2>Blog Topic</h2>
-        <p>Blog Topics are how <?php echo($sitename); ?> includes your posts into the topic areas on the main <?php echo($sitename); ?> site. These topics are for the convenience of the public using the main <?php echo($sitename); ?> site and do not affect your individual blog site in any way.<p> 
+        <h2><?php _e('Blog Topic') ?></h2>
+        <p><?php printf(__('Blog Topics are how %1$s includes your posts into the topic areas on the main %1$s site. These topics are for the convenience of the public using the main %1$s site and do not affect your individual blog site in any way.'), $sitename) ?></p>
         <form name="blogtopicform" action="" method="post">
             <table class="form-table">	
             <tr>
@@ -751,18 +751,18 @@
                     $this->get_topics_select();
             ?>
             <tr>
-            <th scope="row">Blog Sharing Options</th>
+            <th scope="row"><?php _e('Blog Sharing Options') ?></th>
             <td>
             <fieldset>	
-            <legend class="hidden">Blog Sharing Options</legend>
+            <legend class="hidden"><?php _e('Blog Sharing Options') ?></legend>
             <label for="cets_topicexclude_no">
             <input id="cets_topicexclude_no" type="radio" name="cets_topicexclude" value="0" <?php if( $excluded == 0 ) echo " checked"?> />
-            <?php _e("<b>Share my blog's content!</b> - (This will automatically include your article headlines and list your site in the <?php echo($sitename); ?> site directory and on related blog widgets. It can also increase your Google ranking if you haven't blocked search engines.)"); ?>
+            <?php printf(__('<b>Share my blog\'s content!</b> - (This will automatically include your article headlines and list your site in the %1$s site directory and on related blog widgets. It can also increase your Google ranking if you haven\'t blocked search engines.)'), $sitename); ?>
             </label>
 			<br />
             <label for="cets_topicexclude_yes">
 			<input id="cets_topicexclude_yes" type="radio" name="cets_topicexclude" value="1" <?php if( $excluded == 1 ) echo " checked"?> />
-			<?php _e("<b>Hide my blog on the main blog and in widgets.</b> - (Not recommended. This prevents the public from finding your articles and site through the <?php echo($sitename); ?> site directory and widgets. Your site will still be visible to people linking directly to it."); ?>
+			<?php printf(__('<b>Hide my blog on the main blog and in widgets.</b> - (Not recommended. This prevents the public from finding your articles and site through the %1$s site directory and widgets. Your site will still be visible to people linking directly to it.'), $sitename); ?>
             </label>
 			<br />
             </fieldset>
